### DISCUSSION TITLE
Analog pins

### DISCUSSION THREAD

**Discussion by souravlal** [marcel-licence/esp32_midi_sampler#32] [Q&A]
How many analog pins are available on this board the ESP32 audio kit?

**Comment by souravlal** [comment]
Can enyone help me with my above question.👆

**Reply by marcel-licence** [reply]
https://www.makerfabs.com/desfile/files/ESP32-A1S%20Product%20Specification.pdf
look for ADC pins. But it may be better to use an analog multiplexer or external ADC via I2C

**Reply by souravlal** [reply]
I am thinking about it.
